### DISCUSSION TITLE
missing file in binary "cublas64_11.dll"

### DISCUSSION THREAD

**Discussion by ntfrfn** [NVlabs/instant-ngp#1146] [Q&A] [answered]
Hi I tried to run the binary on a windows 10 machine, and it says that "cublas64_11.dll " was not found and that I should reinstall the binary. I don't know what that means. is there something that I can do to get the binary running?

**Comment by Tom94** [comment, marked as answer]
Could you try again with the latest download link? I pushed an attempted fix.

**Reply by ntfrfn** [reply]
Just got back from vacation, and it works right away!
